src/ServiceControl/Recoverability/API/GroupFetcher.cs

```diff
@@ -62,14 +62,19 @@ private static UnacknowledgedRetryOperation[] MapAcksToOpenGroups(FailureGroupVi
                     select unack).ToArray();
         }
 
-        private static FailureGroupView[] GetDBGroups(string classifier, IDocumentSession session)
+        private static FailureGroupView[] GetDBGroups(IDocumentSession session, string classifier, string classifierFilter)
         {
             var groups = session.Query<FailureGroupView, FailureGroupsViewIndex>()
-                .Where(v => v.Type == classifier)
-                .OrderByDescending(x => x.Last)
+                .Where(v => v.Type == classifier);
+
+            if (!string.IsNullOrWhiteSpace(classifierFilter))
+            {
+                groups = groups.Where(v => v.Title == classifierFilter);
```

Comment:
The `Type` is filtered already above.

This is a lucene query, so case is important. The filter value comes from the values that the metrics plugin (and core) provides though, so the case should match between them.